README.md

```diff
@@ -142,8 +142,13 @@ debug('this is hex: %h', new Buffer('hello world'))
 ```
 
 ## Browser support
+  You can build a browser-ready script using [browserify](https://github.com/substack/node-browserify),
```

Comment:
We should mention that it works out of the box with the popular browser bundlers. We mention browserify but at least webpack should be mentioned as well.